cfn.yaml

```diff
@@ -52,3 +52,20 @@ Resources:
       Role: !GetAtt ExecutionRole.Arn
       Runtime: java11
       Timeout: 120
+
+  ScheduledRule:
+    Type: AWS::Events::Rule
+    Properties:
+      ScheduleExpression: cron(20 11 ? * WED *) # MaxMind: "Weekly updates are available every Tuesday"
```

Comment:
[AWS have slightly unusual cron expressions](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-schedule-expressions.html), in that they include a 6th value which is the _year_ of execution, and they use `?` (truth be told, there is quite [a lot of variation](https://en.wikipedia.org/wiki/Cron#CRON_expression) in implementations of cron expressions)

If you get rid of those two idiosyncrasies, you can check this cron with https://crontab.guru/#20_11_*_*_WED :

![image](https://user-images.githubusercontent.com/52038/119473236-e3118700-bd42-11eb-81c4-1e798ccb0fee.png)

So, the expression will execute at 11:20am on Wednesdays, the day after MaxMind release the weekly database update.